src/components/FunctionsPage/Functions.js

```diff
@@ -313,9 +313,13 @@ const Functions = ({
   }
 
   const handleDeployFunctionSuccess = ready => {
-    const { name, tag } = functionsStore.newFunction.metadata
+    let { name, tag } = functionsStore.newFunction.metadata
     const tab = ready === false ? 'logs' : 'overview'
 
+    if (!tag) {
+      tag = 'latest'
+    }
```

Comment:
Since ES12/ES2021, which comes out this year, we can do [this](https://github.com/tc39/proposal-logical-assignment):
```suggestion
    tag ||= 'latest'
```
This should already be supported by Babel.js in plugin `@babel/plugin-proposal-logical-assignment-operators` which is included in `@babel/preset-env`, which we're already using.

Then my only question is, whether you used `||` for nullish values (`null` and `undefined`) or also for the empty string (`''`). If you're only concerned for `null` and `undefined` you can use `??=` instead of `||=`.
```suggestion
    tag ??= 'latest'
```
If `undefined` is the only one you're concerned about, not even `null`, then you can set a default value via destructuring:
```js
let { name, tag = 'latest' } = functionsStore.newFunction.metadata
```